src/hooks/useActiveProject.tsx

```diff
@@ -70,15 +71,19 @@ export const ActiveProjectContextProvider = ({
   const [hookReturnValue, setHookReturnValue] = useState<ActiveProjectProps>(
     {},
   );
+  const { data } = useUser();
+  const [userId, setUserId] = useState(data?.id);
```

Comment:
Might be a little easier to read if we map `data.id` to `userId` and then use a `usePrevious` on that value to see if it changes. That way `userId` will always be the current id.

Reply:
Updated